docs/resources/conversation.md

```diff
@@ -43,6 +43,18 @@ resource "slack_conversation" "test" {
 }
 ```
 
+```hcl
+resource "slack_conversation" "nonadmin" {
+  name              = "my-channel01"
+  topic             = "The channel won't be archived on destroy"
+  permanent_members = []
+  is_private        = true
+  action_on_destroy = "none"
+  
+  
```

Comment:
Remove blank lines

Reply:
Hi Pablo, thank you for accepting the changes.  and Thank you for helping with the code. I just haven't figured out a way to test this option right. I have added the changes and tested them. Please review.